package.json

```diff
@@ -140,6 +140,7 @@
     "eslint-config-airbnb": "^15.1.0",
     "eslint-config-prettier": "^3.1.0",
     "eslint-import-resolver-babel-module": "^5.0.1",
+    "eslint-plugin-fp": "^2.3.0",
```

Comment:
first result with a no-proxy rule. this plugin has some [other rules](https://github.com/jfmengels/eslint-plugin-fp#rules) that might be useful